### DISCUSSION TITLE
Welcome to eXo Community Discussions!

### DISCUSSION THREAD

**Discussion by plamarque** [exoplatform/exo-community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vitorpadilha** [comment]
Hello,

I'm trying create a Space that I all user can auto-join when register in platform, like in builders (that have engage, General and Developers). Have some addon or config to do it? Can you help-me? Thanks.

**Reply by plamarque** [reply]
Here's a way to do it via the UI.
As an administrator, go to Administration > Spaces, find the space that you want all user to auto-join. In the Actions column click the "Space binding" icon (the first) and in the drawer select /platform/users for the group to bind to.

**Comment by serveradmini** [comment]


**Reply by ahamdi** [reply]
@serveradmini Hi and welcome to eXo community
Could you create a new discussion for this issue as this one is for welcoming new community members. And attach the server logs to understand why the server is stopping .
Thanks

**Reply by serveradmini** [reply]
Hello I'm awaiting assistance for https://github.com/exoplatform/exo-community/discussions/31#discussion-5554105

**Comment by serveradmini** [comment]
Hello I'm awaiting assistance for https://github.com/exoplatform/exo-community/discussions/31#discussion-5554105

**Comment by Rajamuthukrishna** [comment]
Hello Team,

I am waiting assistance for https://github.com/exoplatform/exo-community/discussions/44, Kindly anyone help for this..

**Comment by Rajamuthukrishna** [comment]
Hello Team,

I am waiting assistance for https://github.com/exoplatform/exo-community/discussions/76 ,Kindly anyone help for this..

**Comment by simson-m** [comment]
Hi Everyone, Trying integrate the custom login module in eXo Platform with open-idconnect (keycloak). Help me to find the solution for below problem.
1) Facing issue on Login Module that we have build, we don't found any method 'getCurrentHttpServletRequest' on "AbstractLoginModule".
2) We have been using the eXo docker-compose, where to place the jaas conf file to override the default filters
3) In Step 3 for the below link where I can find the configuration file "configuration.xml" to overcome the custom Filter.

https://www.exoplatform.com/blog/guide-to-integrate-linkedin-2-0-connect-with-the-exo-platform-sign-in-mechanism/

**Reply by rdenarie** [reply]
> Facing issue on Login Module that we have build, we don't found any method 'getCurrentHttpServletRequest' on "AbstractLoginModule".

This function is not available in AbstractLoginModule. You can find it in SSOLoginModule : 
https://github.com/Meeds-io/gatein-sso/blob/develop/sso-agent/src/main/java/org/gatein/sso/agent/login/SSOLoginModule.java#L126

You have 2 options : 
- re-implement the function in your login module
- Extends SSOLoginModule

> We have been using the eXo docker-compose, where to place the jaas conf file to override the default filters

jaas.conf file is located in /opt/exo/conf/jaas.conf


> In Step 3 for the below link where I can find the configuration file "configuration.xml" to overcome the custom Filter.

You can inspire from this declaration : 
https://github.com/Meeds-io/push-notifications/blob/develop/service/src/main/resources/conf/portal/configuration.xml#L100-L125

configuration.xml file is put in the jar.

**Comment by z0dium** [comment]
Hi.
I'm trying  to modify locale.properties files. And i'm using docker-compose to start up.
After that i docker exec into exo-exo-1 container and modify files. But changes are not reflected on the fly.
Is there any proper way to reload service to take changes?

**Reply by rdenarie** [reply]
Hi @z0dium Can you register on [eXo Tribe](https://community.exoplatform.com/), and ask you question ? You will have more visibility

**Reply by z0dium** [reply]
I'll do. Thank you

**Reply by z0dium** [reply]
I didnt understand where i should ask questions at eXo Tribe. Could you guide me?

**Reply by rdenarie** [reply]
After loggued on the platform, you have few spaces on the right : 
![image](https://github.com/user-attachments/assets/962bdf53-35d1-4a0b-962e-cb68af915e74)

You can click for example on eXo Lean, and then write a new message in this space, explainin your problem.

Thanks
